collect_app/src/main/res/values-uk/strings.xml

```diff
@@ -595,4 +607,29 @@
   <string name="instance_upload_cancelled">Завантаження екземпляру скасовано</string>
   <string name="hide_old_form_versions_setting_title">Приховати старі версії анкет</string>
   <string name="hide_old_form_versions_setting_summary">Тільки найновіша версія буде відображена в Заповнити анкету</string>
+  <string name="sms_pref_dialog_title">Номер телефону отримувача</string>
+  <string name="sms_pref_title">Номер телефону отримувача</string>
+  <string name="sms_invalid_phone_number">Некоректний номер!</string>
+  <!--Sms submission-->
+  <string name="sms_no_reception">\'Не вдалося\' EEE, MMM dd, yyyy \'в\' HH:mm \'через відсутність отримувача\'</string>
+  <string name="sms_fatal_error">\'Не вдалося\' EEE, MMM dd, yyyy \'в\' HH:mm</string>
+  <string name="sms_airplane_mode">\'Не вдалося\' EEE, MMM dd, yyyy \'в\' HH:mm \'через ввімкнений Режим Без Зв/\'язку\'</string>
+  <string name="sms_no_message">Ця анкета не має питань з позначками повідомлень.</string>
+  <string name="sms_encrypted_message">Зашифровані анкети не можуть бути відправлені як повідомлення.</string>
+  <string name="sms_last_submission_on_date_at_time">\'Остання спроба відправки\' EEE, MMM dd, yyyy \'в\' HH:mm</string>
+  <plurals name="sms_sending">
```

Comment:
Lint is showing an error for the `one` quantity item

https://521-87481171-gh.circle-artifacts.com/0/reports/lint/lint-result.html#ImpliedQuantity